openbb_terminal/API_README.md

```diff
@@ -1,84 +1,280 @@
-you can develop new functionalities and avoid copy-pasting the code from the repository.
+```
 
-**Note:**\
-API keys for data fetching still have to be entered
+<img width="813" alt="Screenshot 2022-10-03 at 23 56 52" src="https://user-images.githubusercontent.com/40023817/193700307-cbb12edc-0a5d-4804-9f3c-a798efd9e69d.png">
 
-### Jupyter Notebook Tricks
+OR
 
-**Get  matplotlib charts in the output cells**\
-If you copy-paste the code below and use it as your initialization then you're matplotlib graphs will be inside
-the result cell.
+```
+from openbb_terminal.api import TerminalStyle
+theme = TerminalStyle("dark", "dark", "dark")
 
-```python
-import matplotlib.pyplot as plt
-import matplotlib_inline.backend_inline
-from openbb_terminal.api import openbb
-%matplotlib inline
-matplotlib_inline.backend_inline.set_matplotlib_formats("svg")
 ```
 
-**Get function signature and docstring**\
-When you press `shift + tab` in jupyter notebook while having the mouse parser in an API function, you get the
-signature and docstring of the function.
 
-### Visual Studio Code Tricks
+<img width="791" alt="Screenshot 2022-10-03 at 23 46 33" src="https://user-images.githubusercontent.com/40023817/193699221-e154995b-653c-40fd-8fc6-a3f8d39638db.png">
+
 
-**Get function docstring and signature**\
-In order to get the docstrings and function signatures for the API when opening a Jupyter Notebook in VSCode,
-you have to install the [Jupyter PowerToys
+
+### 3. Access Documentation
+Each and every command of OpenBB SDK has detailed documentation about input parameters and returned outputs. You can access them in multiple ways:
+
+**Approach 1: Press `shift + tab`**
+This will work out of the box if you're using Jupyter Notebook. In case your IDE is VSCode, you will need to install the [Jupyter PowerToys
 extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.vscode-jupyter-powertoys).
 
-## Code Examples
+<img width="788" alt="Screenshot 2022-10-03 at 23 31 55" src="https://user-images.githubusercontent.com/40023817/193697567-e7143252-c560-441e-84fd-cbe38aeaf0ea.png">
+
+
+**Approach 2: Type `help(command)`**
+
+You can also type `help(command)`, see example below, to see the command' docstring.
+
+<img width="871" alt="Screenshot 2022-10-03 at 23 33 05" src="https://user-images.githubusercontent.com/40023817/193697676-39351008-386d-4c4c-89f2-3de7d8d4e89d.png">
+
+
+**Approach 3: Use OpenBB SDK Documentation page**
+
+Finally, if you prefer to check documentation on a web browser, [OpenBB SDK Documentation](https://openbb-finance.github.io/OpenBBTerminal/api/) will be your best friend. You can browse available commands and search for any specific one that you need.
+
+<img width="1200" alt="Screenshot 2022-10-03 at 18 41 48" src="https://user-images.githubusercontent.com/40023817/193643316-c063df03-4172-487f-ba47-ee60f36a3fef.png">
+
+
+### 4. Set API Keys
+
+TO BE COMPLETED once we finish the development
```

Comment:
@montezdesousa when you finish the development of API key settings, it'd be great to have the documentation here - so users know how to use it! 😄

Reply:
Agree, already on todo list!